src/util/layersBrowser.js

```diff
@@ -467,7 +468,12 @@ L.Control.LayersBrowser = L.Control.Layers.extend({
     }
     this._hideElements(obj, data, layerName, elements); // Filter layer list on initialization
     map.on('moveend', function() { // Update layer list on map movement
-      self._hideElements(obj, data, layerName, elements, true);
+      if (isPageRefreshed < 2 && window.performance.navigation.type !== 1) {
+        isPageRefreshed++;  // Track page moveend events to prevent errors on map.removeLayer when a page is reloaded
+      }
+      if(isPageRefreshed > 1) {
+        self._hideElements(obj, data, layerName, elements, true);
+      }
```

Comment:
The new layer menu removes the map on 'moveend' event if the layer is displayed on the map but is out of bounds. The 'moveend' event was being triggered on page refresh causing the error on the unearthing layer where it couldn't find the layer to remove from the map.

This was weird because that line of code would only run if `map.hasLayer()` returned true. And also I could see the layer displayed on the map and could access it on the console. So I set this up to avoid running that code if it is the first 'moveend' event after a page refresh.

Reply:
Ah okay